recipes/boost/all/dependencies/dependencies-1.90.0.yml

```
@@ -0,0 +1,292 @@
+configure_options:
+- atomic
+- charconv
+- chrono
+- cobalt
+- container
+- context
+- contract
+- coroutine
+- date_time
+- exception
+- fiber
+- filesystem
+- graph
+- graph_parallel
+- iostreams
+- json
+- locale
+- log
+- math
+- mpi
+- nowide
+- process
+- program_options
+- python
+- random
+- regex
+- serialization
+- stacktrace
+- test
+- thread
+- timer
+- type_erasure
+- url
+- wave
+dependencies:
+  atomic: []
+  charconv: []
+  chrono: []
+  cobalt:
+  - container
+  - context
+  cobalt_io:
+  - cobalt
+  cobalt_io_ssl:
+  - cobalt
+  container: []
+  context: []
+  contract:
+  - exception
+  - thread
+  coroutine:
+  - context
+  - exception
+  date_time: []
+  exception: []
+  fiber:
+  - context
+  - filesystem
+  fiber_numa:
+  - fiber
+  filesystem:
+  - atomic
+  graph:
+  - math
+  - random
+  - regex
+  - serialization
+  graph_parallel:
+  - filesystem
+  - graph
+  - mpi
+  - random
+  - serialization
+  iostreams:
+  - random
+  - regex
+  json:
+  - container
+  locale:
+  - charconv
+  - thread
+  log:
+  - atomic
+  - date_time
+  - exception
+  - filesystem
+  - regex
+  - thread
+  log_setup:
+  - log
+  math: []
+  math_c99:
+  - math
+  math_c99f:
+  - math
+  math_c99l:
+  - math
+  math_tr1:
+  - math
+  math_tr1f:
+  - math
+  math_tr1l:
+  - math
+  mpi:
+  - graph
+  - serialization
+  mpi_python:
+  - mpi
+  - python
+  nowide:
+  - filesystem
+  numpy:
+  - python
+  prg_exec_monitor:
+  - test
+  process:
+  - filesystem
```

```suggestion
  - filesystem
  - context
```

Related to https://github.com/conan-io/conan-center-index/pull/26752

No new changes were included in Boost Process that are affect that dependency: https://github.com/boostorg/process/compare/boost-1.89.0...boost-1.90.0#files_bucket